pages/ProfilePage.tsx

```diff
@@ -0,0 +1,86 @@
+import Head from "next/head";
+import ChooseRideButton from "../components/chooseRideButton/chooseRideButton";
+import HeaderButton from "../components/headerButton/headerButton";
+import SubHeader from "../components/subHeader/subHeader";
+import styles from "../styles/ProfilePage.module.css";
+import Calendar from "react-calendar";
+import "react-calendar/dist/Calendar.css";
+import FooterButton from "../components/footerButton/footerButton";
+
+export default function LoginPage() {
+  return (
+    <div className={styles.container}>
+      <Head>
```

Comment:
you can put this header in the  _app.tsx so you don't have to import in on every page